CHANGES.md

```diff
@@ -1,7 +1,10 @@
-
 master
 ======
 
+- global variables from modules referenced in sys.modules in the child process
+  now overrides the initial global variables of the pickled function.
+  ([issue #187](https://github.com/cloudpipe/cloudpickle/issues/187))
```

Comment:
I would rephrase this as:

"""
Ensure that unpickling a locally defined function that accesses the global variables of a module does not reset the values of the global variables if they are already initialized.
"""